src/ring_theory/integrally_closed.lean

```diff
@@ -124,8 +129,32 @@ namespace integral_closure
 
 open is_integrally_closed
 
-variables {R : Type*} [comm_ring R] [is_domain R]
-variables (K : Type*) [field K] [algebra R K] [is_fraction_ring R K]
+variables {R : Type*} [comm_ring R]
+variables (K : Type*) [field K] [algebra R K]
+
+theorem frange_subset_integral_closure
+  {f : R[X]} (hf : f.monic) {g : K[X]} (hg : g.monic) (hd : g ∣ f.map (algebra_map R K)) :
+  (g.frange : set K) ⊆ (integral_closure R K).to_subring :=
```

Comment:
Seems like this could also be stated using `lifts`, but is it more convenient that way? Seems the proof of `coeff_mem_subring_of_splits` was made longer by the change. I wasn't aware of `lifts` when I wrote these lemmas (I learned it [here on Zulip](https://leanprover.zulipchat.com/#narrow/stream/217875-Is-there-code-for-X.3F/topic/.60exists_monic_map_eq_of_mem_range.60/near/291676494)), so the proofs could be unnatural to adapt to the `lifts` API.